src/test/java/de/tum/in/www1/artemis/config/WebsocketConfigurationTest.java

```diff
@@ -0,0 +1,29 @@
+package de.tum.in.www1.artemis.config;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Set;
+
+import org.junit.jupiter.api.Test;
+
+import de.tum.in.www1.artemis.AbstractSpringIntegrationBambooBitbucketJiraTest;
+import de.tum.in.www1.artemis.config.websocket.WebsocketConfiguration;
+
+class WebsocketConfigurationTest extends AbstractSpringIntegrationBambooBitbucketJiraTest {
+
+    @Test
+    void testGetTopicRelayPrefixes() {
+        // No special profiles activated -> Do not forward quiz messages
+        assertThat(WebsocketConfiguration.getTopicRelayPrefixes(Set.of(""))).containsExactlyInAnyOrder("/topic");
+
+        // Only decoupling profile -> Forward quiz messages
+        assertThat(WebsocketConfiguration.getTopicRelayPrefixes(Set.of("decoupling"))).containsExactlyInAnyOrder("/topic", "/queue/quizExercise");
+
+        // Only quiz profile -> Do not forward quiz messages (because decoupling profile is not enabled)
+        assertThat(WebsocketConfiguration.getTopicRelayPrefixes(Set.of("quiz"))).containsExactlyInAnyOrder("/topic");
+
+        // Quiz & lecture profile -> Do not forward quiz messages
```

Comment:
I guess it should be decoupling and quiz profile in the comment

Reply:
Thanks, yes!